TrafficCapture/dockerSolution/src/main/docker/elasticsearchWithSearchGuard/Dockerfile

```diff
@@ -16,7 +16,6 @@ RUN sed 's/searchguard/plugins.security/g' $ELASTIC_SEARCH_CONFIG_FILE |  \
 # The following two commands are more convenient for development purposes,
 # but maybe not for a demo to show individual steps
 RUN /root/enableTlsConfig.sh $ELASTIC_SEARCH_CONFIG_FILE
-# Do this to disable HTTP auth
```

Comment:
Not a big deal, but why was this comment removed? Is it no longer accurate?

Reply:
Thanks to both of you